front/src/api/todos-api.ts

```diff
@@ -22,8 +22,14 @@ export const todosAPI = {
         return todo;
       })
   },
-  updateTodo(id: number, newTitle: string) {
-    return axiosInstance.put<Todo>(`/todos/${id}`, { title: newTitle })
+  updateTodo(id: number, newTitle: string, newImportance: string, newIsDone: boolean) {
+    return axiosInstance.put<Todo>(
```

Comment:
here you will have merge conflict with PR !14 because it updates the same block
better to avoid doing related changes in parallel branches